packages/shared/src/contexts/PaymentContext.tsx

```diff
@@ -100,6 +100,15 @@ export const PaymentContextProvider = ({
               event?.data.currency_code,
               event?.data?.transaction_id,
             );
+            if ('gifter_id' in event.data.custom_data) {
+              logRef.current({
+                event_name: LogEvent.GiftSubscription,
```

Comment:
This should be the `complete gift subscription` event, isn't it?
Let's include all the other fields like the other event.
Also let's do if it's a gift we send only this one, otherwise we send the previous. Basically tertiary condition on event_name